src/app/components/containers/about.rs

```diff
@@ -0,0 +1,46 @@
+use crate::{
+    app::context::{Context, SidebarStage, Stage},
+    model::Model,
+};
+use ratatui::{
+    layout::Rect,
+    style::{Color, Style},
+    text::Text,
+    widgets::{Block, Borders, Paragraph},
+    Frame,
+};
+
+pub struct About {}
+
+impl About {
+    pub fn new() -> About {
+        About {}
+    }
+
+    pub fn render(&self, props: AboutProps, frame: &mut Frame, area: Rect) {
+        let color = if props.on { Color::Green } else { Color::White };
```

Comment:
Let's destruct the props object before we use it to account for future scalability.

Reply:
Done.